temporal-sdk/src/main/java/io/temporal/worker/Worker.java

```diff
@@ -181,7 +181,7 @@ private static SingleWorkerOptions toLocalActivityOptions(
       List<ContextPropagator> contextPropagators,
       Scope metricsScope) {
     return toSingleWorkerOptions(factoryOptions, options, clientOptions, contextPropagators)
-        .setPollerOptions(PollerOptions.newBuilder().build())
```

Comment:
ActivityWorker and WorkflowWorker PollerOptions specify this parameter explicitly. 
To simplify the untangling for the reader, let's specify it for LocalActivityWorker explicitly too. 1 is the default value for pollThreadCount used inside PollerOptions.